src/Components/LandingPage/sections/MostPickedSection.tsx

```diff
@@ -0,0 +1,195 @@
+import img1 from '../../../assets/download.png'
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import { Box, FormControl, Grid, TextField } from '@mui/material';
+import { useContext, useEffect, useState } from "react";
+import styles from './MostPickedTaskTest.module.css'
+import RemoveRedEyeIcon from '@mui/icons-material/RemoveRedEye';
+import FavoriteBorderOutlinedIcon from '@mui/icons-material/FavoriteBorderOutlined';
+import axios from 'axios';
+import { AuthContext } from '../../../Context/AuthContext';
+import Modal from '@mui/material/Modal';
+import { useForm } from 'react-hook-form';
+import { Link, useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+
+
+export default function MostPickedSection() {
+
+  const { userRole }: any = useContext(AuthContext)
+  const [showState, setShowState] = useState();
+  const handleClose = () => setShowState(false);
+  const navigate = useNavigate()
+  const [ads, setAds] = useState();
+
+  const getMostPicked = () => {
```

Comment:
Don't forget to add loading state for this API to indicate to users that the rooms are currently being loaded.